postgres/src/main/java/org/jdbi/v3/postgres/PostgresTypes.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.jdbi.v3.core.internal.exceptions.Unchecked;
+import org.postgresql.PGConnection;
+import org.postgresql.util.PGobject;
+
+/**
+ * Handler for PostgreSQL custom types.
+ */
+public class PostgresTypes {
```

Comment:
Should this be a `JdbiConfig` class instead of static singleton?

Reply:
Looking at this again, I think it _should_ be a JdbiConfig class.

We'll have to add a new method overload `JdbiPlugin.customizeConnection(Connection, ConfigRegistry)` to give `PostgresPlugin.customizeConnection()` access to the config registry.